### DISCUSSION TITLE
Rocoliilil/17

### DISCUSSION THREAD

**Discussion by giscus** [GDSC-Daejin/gdsc-dju-blog-fe#57] [Announcements]
# Rocoliilil/17

 Google Developers에서 제공하는 프로그램으로, GDSC KR는 21명의 Lead를 주축으로 21개 대학교에 선발되었습니다.

http://localhost:3000/Rocoliilil/17

**Comment by HyeokE** [comment]
안녕

**Reply by gudcks0305** [reply]
## 답글달아버리기~
